### PR TITLE
Add cog to play spooky sounds

### PR DESCRIPTION
**Pull request by meooow25**
For issue #28 

A SpookySound cog plays a spooky sound when the command `.spookysound` is used.
The bot connects to a voice channel, plays the sound, and disconnects. The command cannot be used more than once per two minutes.

### CODE REVIEW

**Comment by jodth07** [comment]
We should have the spooky sounds attached to a command, or listening to a set of words. See the implementation of spookey react. You can still keep it limited to once, every 2 mins though.

**Comment by meooow25** [comment]
I'm not sure what you mean, since the behavior is exactly as described in issue #28. Perhaps I should have used a better PR description, updated now.

File: bot/cogs/spookysound.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import random
+from pathlib import Path
+
+import discord
+from discord.ext import commands
+
+HACKTOBERBOT_VOICE_CHANNEL_ID = 101010  # Replace with actual channel ID
```
Comment (heavysaturn) [inline bot/cogs/spookysound.py]: before we can merge this, this should actually be replaced.

```suggestion
HACKTOBERBOT_VOICE_CHANNEL_ID = 498804789287714816
```

File: bot/cogs/spookysound.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+
+class SpookySound:
```
Comment (heavysaturn) [inline bot/cogs/spookysound.py]: Can we get a class docstring?

File: bot/cogs/spookysound.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        """
+        Connect to the Hacktoberbot voice channel, play a random spooky sound, then disconnect. Cannot be used more than
+        once in 2 minutes.
+        """
+        await ctx.send("Initiating spooky sound...")
+        voice = await self.channel.connect()
+        file_path = random.choice(self.sound_files)
+        src = discord.FFmpegPCMAudio(str(file_path.resolve()))
+
+        async def disconnect():
```
Comment (heavysaturn) [inline bot/cogs/spookysound.py]: I think this should just be a helper method that takes the voice object as a parameter. It's not ideal to define helper functions inside of methods.

File: bot/cogs/spookysound.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (meooow25) [reply bot/cogs/spookysound.py]: Updated.

Comment (meooow25) [reply bot/cogs/spookysound.py]: Updated.

File: bot/cogs/spookysound.py (hunk 2 above)
Comment (meooow25) [reply bot/cogs/spookysound.py]: Done.

File: bot/cogs/spookysound.py (hunk 3 above)
Comment (meooow25) [reply bot/cogs/spookysound.py]: Thanks, I was not aware of that. I have changed it to a static method.

**Comment by meooow25** [comment]
Thanks, I have made the requested changes.

File: Pipfile [hunk 4]
```diff
@@ -5,6 +5,7 @@ name = "pypi"
 
 [packages]
 "discord.py" = {ref = "rewrite", git = "https://github.com/Rapptz/discord.py"}
```
Comment (sco1) [inline Pipfile]: The `discord.py` requirement will need to be changed to:

```python
discord-py = {extras = ["voice"], ref = "rewrite", git = "https://github.com/Rapptz/discord.py"}
```

`PyNaCl` is already a requirement for this so it doesn't need to be added to our Pipfile

Comment (ByteCommander) [reply Pipfile]: @meooow25 Could you please make that requested change so that your PR is ready to be merged?

Comment (meooow25) [reply Pipfile]: Apologies for the delay, I am having trouble with locking the pipenv due to incompatible dependencies. I asked about it in the Discord server but received no solution.

`Could not find a version that matches pycodestyle<2.4.0,<2.5.0,<3.0.0,>=2.0.0,>=
2.4.0`

`pycodestyle < 2.4.0` and `>= 2.4.0` is of course impossible to satisfy. Some of the `flake8` packages are probably causing this. What should be done here?

Comment (sco1) [reply Pipfile]: This is some other issue with your environment, I just downloaded your branch and it installs & locks just fine.

We'll have to do some more research to figure out what's going on.

**Pull request merged**